src/governance.ts

```diff
@@ -158,7 +217,7 @@ export class GovernanceCanister {
    */
   public claimOrRefreshNeuronFromAccount = async (
     request: ClaimOrRefreshNeuronFromAccount
-  ): Promise<NeuronId> => {
+  ): Promise<NeuronId | TransferError> => {
```

Comment:
ditto regarding error types. We can either define a `ClaimOrRefreshNeuronFromAccountError`, or return a `NeuronId?` since the only error here is that the neuron wasn't found.

Reply:
ok, error per method. I could just return `undefined` if not found and return a NotFound.